packages/vuetify/src/util/helpers.ts

```diff
@@ -378,11 +378,14 @@ export function filterInputAttrs (attrs: Record<string, unknown>) {
 
 /**
  * Returns the set difference of B and A, i.e. the set of elements in B but not in A
+ * A and B can be sets, but the function always returns the result as an array.
```

Comment:
@KaelWD What do you think of this change?

Reply:
If they're already sets .has should be faster than .includes

Reply:
Refactored to use `symmetricDifference` instead

Reply:
We can't use these yet unfortunately: https://caniuse.com/mdn-javascript_builtins_set_symmetricdifference